laboratory-service/src/main/java/com/raf/si/laboratoryservice/mapper/ReferralMapper.java

```diff
@@ -0,0 +1,67 @@
+package com.raf.si.laboratoryservice.mapper;
+
+import com.raf.si.laboratoryservice.dto.request.CreateReferralRequest;
+import com.raf.si.laboratoryservice.dto.response.ReferralListResponse;
+import com.raf.si.laboratoryservice.dto.response.ReferralResponse;
+import com.raf.si.laboratoryservice.model.Referral;
+import org.apache.catalina.mapper.Mapper;
+import org.springframework.data.domain.Page;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
```

Comment:
U maperima ces samo za metode koje request prebacuju u model morati da promenis da prebace iz stringa u enum, pogledaj kako je to radjeno u maperima u patient servisu